core/linker/default.h

```diff
@@ -54,7 +54,10 @@ SECTIONS
 {
     .text :
     {
+        __code_start__ = .;
         *(.text.main_entry)
+        . = __code_start__ + 0x100;
+        KEEP(*(.rodata.uvisor_privcall))
```

Comment:
@Patater : Please put this structure right before the start of uvisor config (with a length indicator to seek back and a magic value + the version value mentioned in earlier code)